docs/02.API-REFERENCE.md

```diff
@@ -101,6 +101,39 @@ created by API functions has the error flag set.
 typedef uint32_t jerry_value_t;
 ```
 
+## jerry_context_data_header_t
+
+**Summary**
+
+Structure that holds a single user context item, identified by its manager. It is intended to serve as the base for
+larger structures that need to be allocated at runtime.
+
+**Prototype**
+
+```c
+typedef struct jerry_context_data_header
+{
+  struct jerry_context_data_header *next_p; /**< pointer to next context item */
+  const struct jerry_context_data_manager *manager_p; /**< manager responsible for deleting this item */
+} jerry_context_data_header_t;
```

Comment:
Also, if we add the number of bytes needed as a field of manager_t then we don't need to call init_cb twice.